specs/IR-0006/suite/test_spec_ir0006.py

```
@@ -0,0 +1,174 @@
+        fm, _ = _fm_text()
+        for key in ("inv", "budget", "decision"):
+            block = _fm_block(fm, key)
+            self.assertGreaterEqual(len(block.strip()), 40, f"{key} 块过薄")
+        self.assertIn("INV-01", _fm_block(fm, "inv"))
+        self.assertGreaterEqual(
+            len(re.findall(r"DECISION-\d+", _fm_block(fm, "decision"))), 6,
+            "decision 块不足 6 条")
+
+    def test_nongoals(self):
+        fm, _ = _fm_text()
+        items = [l for l in _fm_list_block(fm, "nonGoals").splitlines() if l.startswith("- ")]
```

Ruff (0.16.2)

[error] Ambiguous variable name: `l`

(E741)